babelsberg/constraintinterpreter.js

```diff
@@ -509,7 +508,7 @@ Object.subclass('Constraint', {
     addConstraintVariable: function(v) {
         if (v && !this.constraintvariables.include(v)) {
             this.constraintvariables.push(v);
-            v.logTimings = this.logTimings;
+            v.logTimings = this._options.logTimings;
```

Comment:
you could take a look at the reevaluation branch, see what I did there to store the options and compare that to the _options solution